imap_processing/hi/l1c/hi_l1c.py

```diff
@@ -428,3 +446,23 @@ def number_of_products(self) -> int:
             calibration product definitions.
         """
         return len(self._obj.index.unique(level="cal_prod_num"))
+
+
+def coincidence_type_string_to_int(coincidence_type_str: str) -> int:
+    """
+    Convert a coincidence type string to a coincidence type integer value.
+
+    Parameters
+    ----------
+    coincidence_type_str : str
+        The coincidence type string containing the list of detectors hit.
+        e.g. "AC1C2".
```

Comment:
It'd be helpful to also comment what this example should output. I think this would be  `2**3 + 2**1 + 2**0` = 11, right?